day2/question2/solution2.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+)
+var wg sync.WaitGroup
+func add(sum *int, rating int){
+	*sum+=rating
```

Comment:
add a mutex lock

Reply:
added a mutex and handled the concurrency